app/scripts/vendor/coseme-client/client.js

```diff
@@ -241,33 +241,49 @@
 
       function onSubjectUpdated(senderJID, timestamp, messageId,
                                 subject, displayName, author) {
-        fire('notification', pub,
-          newNotification('group-subject', messageId, senderJID,
-                          { subject: subject }, displayName, author)
-        );
+        fire('notification', pub, newNotification(
+          'group-subject',
+          messageId,
+          senderJID,
+          { subject: subject },
+          timestamp,
+          displayName,
+          author
+        ));
         methods.call('notification_ack', [senderJID, messageId]);
       }
 
       function onGroupParticipantEvent(event, senderJID, jid, _,
                                        timestamp, messageId) {
-        fire('notification', pub,
-          newNotification('group-participant', messageId, senderJID,
-                          { event: event, participant: jid.split('@')[0] })
-        );
+        fire('notification', pub, newNotification(
+          'group-participant',
+          messageId,
+          senderJID,
+          { event: event, participant: jid.split('@')[0] },
+          timestamp
+        ));
         methods.call('notification_ack', [senderJID, messageId]);
       }
 
       function onGroupPictureEvent(event, senderJID, timestamp, messageId,
                                    pictureId, author) {
-        fire('notification', pub,
-          newNotification('group-picture', messageId, senderJID,
-                          { event: event, pictureId: pictureId }, null, author)
-        );
+        fire('notification', pub, newNotification(
+          'group-picture',
+          messageId,
+          senderJID,
+          { event: event, pictureId: pictureId },
+          timestamp,
+          null,
+          author
+        ));
         methods.call('notification_ack', [senderJID, messageId]);
       }
 
-      function newNotification(type, messageId, senderJID,
-                               content, displayName, author) {
+      function newNotification(type, messageId, senderJID, content,
```

Comment:
so we use here a timestamp in the last position. I prefer this behaviour when adding new fields to a method signature